content/post/2017-05-29-romain-françois-s-site-with-blogdown-hugo-rbind.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Romain François’s site with blogdown,hugo,rbind
+author: Romain François
+date: '2017-05-29'
+slug: romain-françois-s-site-with-blogdown-hugo-rbind
```

Comment:
It seems that the unicode character `ç` works fine in the URL of this post, but it might be a good idea to use pure ASCII (perhaps also make the slug shorter, e.g. `romain-site`). It is up to you. I'm totally fine no matter if you want to change it or not.